Index: src/test/resources/features/us_07/US_07_TC_01.feature
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/test/resources/features/us_07/US_07_TC_01.feature b/src/test/resources/features/us_07/US_07_TC_01.feature
deleted file mode 100644
--- a/src/test/resources/features/us_07/US_07_TC_01.feature	
+++ /dev/null	
@@ -1,10 +0,0 @@
-Feature: Get Cucumber Projesi
-
-  Scenario: Arif Senerio
-    Given herokuapp base url e "booking" ve 19 ekle
-    When User sends a GET Request to the url
-    Then HTTP Status Code should be 200
-    And  Content Type should be JSON
-    And  Status Line should be "HTTP/1.1 200 OK"
-
-
